opt_PXL.ino

```diff
@@ -114,47 +118,57 @@ void setup() {
 void loop() {
   switch (menuItem) {
     case 1:
-      display.begin(16,2);
+      display.setTextSize(1);
+      display.setTextColor(WHITE);
+      display.setCursor(0, 8);
       display.println( F("1:File Select   "));
-      display.setCursor(0, 1);
+      display.setCursor(0, 14);

```

Comment:
Nitpick: `14` is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) - consider turning into a const to indicate meaning.